server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/UaaConfigurationTokenKeyTest.java

```diff
@@ -0,0 +1,32 @@
+package org.cloudfoundry.identity.uaa.impl.config;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class UaaConfigurationTokenKeyTest {
+
+  private YamlConfigurationValidator<UaaConfiguration> validator = new YamlConfigurationValidator<UaaConfiguration>(
+      new UaaConfiguration.UaaConfigConstructor());
+
+  private UaaConfiguration createValidator(final String yaml) {
+    validator.setExceptionIfInvalid(true);
+    validator.setYaml(yaml);
+    validator.afterPropertiesSet();
+    return validator.getObject();
+  }
+
+  @Test
+  void testTokenKey() {
+    assertNotNull(createValidator("uaa:\n" +
```

Comment:
It doesn't feel right to have this large input to the `createValidator` function, and the only thing we test is that the output is not null. Could we test a little more?

Reply:
This part is outdated

Reply:
@strehle and I confirmed this is adjusted.